helix-term/src/commands/lsp.rs

```diff
@@ -1065,9 +1066,11 @@ where
 
     cx.callback(
         future,
-        move |editor, compositor, response: Option<lsp::GotoDefinitionResponse>| {
-            let items = to_locations(response);
-            goto_impl(editor, compositor, items, offset_encoding);
+        move |editor, compositor, response: Option<lsp::GotoDefinitionResponse>| match to_locations(
```

Comment:
The `match` is concise but it looks a bit awkward to me. Style-wise a regular `if` seems clearer to me:

```rust
let items = to_locations(response);
if items.is_empty() {
    editor.set_error("No definitions found.");
} else {
    goto_impl(editor, compositor, items, offset_encoding);
}
```

and same for the change for references below

Reply:
I had this earlier, but switched to using a `match`. But I agree. Will revert to using a plain `if-else`.